package.json

```diff
@@ -1,18 +1,20 @@
 {
   "name": "dtrace-provider",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
Shouldn't this be a minor version bump instead of a patch version bump? It seems these changes add to the API, and so a downgrading from 0.7.1 to 0.7.0 would break if a user was taking advantage of the new way to pass callbacks' parameters.